### PR TITLE
Feature/save temps

### PR DESCRIPTION
**Pull request by jmikedupont2**
saves the cuda temp dirs, 
this needs more work to turn the flags on and off as well, but am sharing just in case someone wants this.

### CODE REVIEW

**Comment by jmikedupont2** [comment]
It needs more cases, only works for compiler id right now

**Comment by jmikedupont2** [comment]
ok now i have a large bin ./build/nvcc_tmp/ggml-cuda.compute_70.sm_70.cubin: ELF 64-bit LSB executable, NVIDIA CUDA architec\ture,, statically linked, with debug_info, not stripped

**Comment by cebtenzzre** [comment]
You're going to have to remove all of the unrelated changes if you want it to be considered for merging. Also, what is the benefit of saving the temporary files? Faster incremental compile times?

**Comment by jmikedupont2** [comment]
Yes, this is still work in progress and needs more cleanup. the point here
is to get disassmbly and map files for usage with nsys to get fine tuned
output of performance on a query by query basis eventually.

File: models/ggml-vocab-aquila.gguf
Comment (cebtenzzre) [inline models/ggml-vocab-aquila.gguf:1]: I don't think you meant to delete these. They're needed for the tokenizer tests.

**Comment by jmikedupont2** [comment]
Yes sorry about that it got deleted when I was working on the lfs just
ignore those changes.

File: models/ggml-vocab-aquila.gguf
Comment (jmikedupont2) [reply models/ggml-vocab-aquila.gguf:1]: thanks for pointing this out, i rebased and removed the lfs. the large data files have moved to https://huggingface.co/datasets/h4/llama.cpp/tree/master

**Pull request closed without merging**